components/credentials/build.gradle

```diff
@@ -68,6 +70,7 @@ dependencies {
     implementation("org.apache.commons:commons-lang3:${apacheCommonsLang3Version}")
     implementation("org.apache.httpcomponents:httpclient:${apacheHttpClientVersion}")
     implementation("commons-codec:commons-codec:${commonsCodecVersion}")
+    implementation("com.google.protobuf:protobuf-java:3.25.2")
```

Comment:
The same as above. I'd like to understand how we got to the point of adding this dependency here to resolve the issue. However, it apparently resolved the issue, so I am fine with it.